Python/crossfire/crossfire/occurrences.py

```diff
@@ -0,0 +1,26 @@
+from queue import Queue, Empty
+import random
```

Comment:
É um padrão fazer os `import`s absolutos primeiro e depois os “parciais”
```suggestion
import random
from queue import Queue, Empty
```

Algumas bibliotecas como [`isort`](https://pypi.org/project/isort/) ajudam a manter esses padrões de estilo. Se quiser, posso adicionar à CI ; )
```

Reply:
> Algumas bibliotecas como [`isort`](https://pypi.org/project/isort/) ajudam a manter esses padrões de estilo. Se quiser, posso adicionar à CI ; )

Massa, não sabia disso.
Posso ver de adicionar isso à CI e tomar mais cuidado.